### ISSUE TITLE
Containing data inside a `WASM`

### ISSUE THREAD

**Issue by shkup** [open-policy-agent/opa#6809]
When I create a regular bundle I can contain there both data and policy.
I would like to request a similar option when creating a `wasm` file. 
Sometimes the data is a `json` file that reading it, parsing it and set it by a `loadData` call is cumbersome and costly in terms of performance. 
Also, it will be nice if the `build -t wasm`command will create a wasm directly rather a bundle file that you need to extract the `wasm` file from (Not the end of the work though but I think makes sense that this is like a "compilation" product).

**Comment by anderseknert** [comment]
> Also, it will be nice if the build -t wasmcommand will create a wasm directly rather a bundle file that you need to extract the wasm file from (Not the end of the work though but I think makes sense that this is like a "compilation" product).

Agreed, same for `-t plan`. If we can avoid users having to repeat extra steps in these flows, we should.
